.github/workflows/release.yml

```
@@ -0,0 +1,90 @@
+name: ReleaseCmdStanPy
+
+on:
+  workflow_dispatch:
+    inputs:
+      new_version:
+        description: 'New version, for example: 0.9.69'     
+        required: true
+      old_version:
+        description: 'Old version, for example: 0.9.68'
+        required: true
+
+jobs:
+  release-cmdstanpy:
+    name: publish release and update read the docs default version
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python-version: [3.9]
+      fail-fast: false
+    steps:
+      - name: Check out source code
+        uses: actions/checkout@v2
+      
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install dependencies (python)
+        run: |
+          pip install --upgrade pip wheel twine codecov sphinx sphinx_rtd_theme requests
+          pip install -r requirements.txt
+
+      - name: Setup git identity
+        run: |
+          git config --global user.name "github-actions[bot]"
+          git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
+
+      - name: Create release branch
+        run: |
+          git checkout -b release/v${{ github.event.inputs.new_version }}
+
+          sed -i 's/${{ github.event.inputs.old_version }}/${{ github.event.inputs.new_version }}/g' cmdstanpy/_version.py
```

```suggestion
          sed -i 's/^__version__ = .*$/__version__ = '${{ github.event.inputs.new_version }}'/g' cmdstanpy/_version.py
```
this could be done with regular expression

---

This won't need the `old_version` at all right ?

---

That's very nice!! The only issue with it, is that it will remove the `'`, at least it did for me locally

---

Ok, it did work for me.. or did it?

Can we use " for outer parts?

---

Sure we can, I'll sort it out in a bit! Had to jump on a call and forgot to come back to this.

---

This should probably work, I will update the original msg

    '\''

and then the result is `'`

---

Changes are now pushed.
Thank you for the valuable feedback!